src/main/java/Duke.java

```diff
@@ -1,10 +1,104 @@
+import java.util.Arrays;
+import java.util.Scanner;
```

Comment:
Maybe should leave an empty line after all the imports?

Reply:
Will space out between the imports and main function in the next push